app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -15,8 +15,11 @@ import {
   removeTestFromDraft,
   addTestToDraft,
   toggleDraftLabOrdersUgency,
+  addDraftLabOrders,
+  deleteDraftLabOrder,
 } from '../../actions/draftLabOrderAction';
```

Comment:
You are not using these two actions you are importing anywhere in this file `addDraftLabOrders`, `deleteDraftLabOrder`, if you do not intend to use it, I suggest you remove it.

Reply:
fixed